java/src/main/java/com/powsybl/python/TemporaryCurrentLimitContext.java

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.python;
+
+import com.powsybl.iidm.network.Branch;
+
+import java.util.Objects;
+
+/**
+ * @author Etienne Lesot <etienne.lesot at rte-france.com>
+ */
+public class TemporaryCurrentLimitContext {
```

Comment:
Just naming: I know it's already used in some other places, but "context" is not really appropriate, maybe we could use "data" instead.

Context is more appropriate when you define a kind of environement, for ex. a set of data, that will be used for a specific purpose: for example SecurityAnalysisContext makes sense, because we define the context in which the security analysis will be run.